sixtyfps_compiler/typeloader.rs

```diff
@@ -15,7 +15,7 @@ use std::rc::Rc;
 
 use crate::diagnostics::{BuildDiagnostics, Spanned};
 use crate::object_tree::{self, Document};
-use crate::parser::{syntax_nodes, NodeOrToken, SyntaxKind, SyntaxToken};
+use crate::parser::{normalize_identifier, syntax_nodes, NodeOrToken, SyntaxKind, SyntaxToken};
```

Comment:
The convention in use usually is not to `use` functions, but to qualify them.

Reply:
Ah, makes sense. I thought because the same was done for `identifier_text` it was okay, but what you say makes more sense for a code readability point of view. I'll change both in a separate commit after landing.